src/emscripten-optimizer/istring.h

```diff
@@ -59,7 +59,7 @@ struct IString {
     }
   };
 
-  IString() : str(nullptr) {}
+  IString() {}
```

Comment:
Not sure it makes a big difference, but I generally see `= default` used for cases like this.

Reply:
Thanks, yeah, we should do that too. Adding a commit now.